app/views/componentsP5/table_1/ucr.scala.xml

```diff
@@ -1,23 +1,23 @@
 @(ucr: String)
-<fo:table-cell  alignment-adjust="auto" border-style="solid" border-width="1pt" border-color="black">
-  <fo:block>
-    <fo:table width="100%" table-layout="fixed">
-      <fo:table-column/>
-      <fo:table-column/>
-      <fo:table-body>
-        <fo:table-row height="2mm">
-          <fo:table-cell number-columns-spanned="1">
-            <fo:block margin-left="0.5mm">UCR</fo:block>
-          </fo:table-cell>
-        </fo:table-row>
-        <fo:table-row number-columns-spanned="4" height="6mm">
-          <fo:table-cell margin-left="0.5mm">
-            <fo:block>
-              @ucr
-            </fo:block>
-          </fo:table-cell>
-        </fo:table-row>
-      </fo:table-body>
-    </fo:table>
-  </fo:block>
+<fo:table-cell alignment-adjust="auto" border-style="solid" border-width="1pt" border-color="black">
+    <fo:block>
+        <fo:table width="100%" table-layout="fixed">
+            <fo:table-column/>
+            <fo:table-column/>
+            <fo:table-body>
+                <fo:table-row height="2%">
+                    <fo:table-cell number-columns-spanned="1">
+                        <fo:block margin-top="3%" margin-left="2%">UCR</fo:block>
```

Comment:
These values seem small? Does 2mm == 2% (I havent seen the output so it may be the case)

Reply:
See picture above